### PR TITLE
Add `max_age` parameter to `decode_token` method

### PR DESCRIPTION
**Pull request by TheDoubleK**
We want to have different ages for different types of token (invite vs password-reset).  We can do that if we add this parameter.

### CODE REVIEW

File: dmutils/email.py [hunk 1]
```diff
@@ -57,12 +57,12 @@ def generate_token(data, secret_key, salt):
     return ts.dumps(data, salt=salt)
 
 
-def decode_token(token, secret_key, salt):
+def decode_token(token, secret_key, salt, max_age=86400):
```
Comment (quis) [inline dmutils/email.py]: Could you name this `max_age_in_seconds`, if the unit is seconds?

Or call the function with a sensible unit, eg `decode_token(max_age_in_hours=48)` and calculate the seconds inside the function?

Comment (robyoung) [inline dmutils/email.py]: I would have said max-age as a number of seconds is pretty common on the web (HTTP cache headers, cookie lifetimes). I'd rather it stays as seconds as it's then more instantly comparable to these other numbers.

**Comment by TheDoubleK** [comment]
Yes - better to make the units explicit.  I think sticking with seconds as the unit is good (even if it means using some big numbers).

**Comment by minglis** [comment]
👍

**Pull request merged**